frontend/src/metabase/components/LogoIcon.jsx

```diff
@@ -5,25 +5,20 @@ import cx from "classnames";
 import { PLUGIN_LOGO_ICON_COMPONENTS } from "metabase/plugins";
 
 class DefaultLogoIcon extends Component {
-  static defaultProps = {
-    size: 32,
-  };
-
   static propTypes = {
-    size: PropTypes.number,
     width: PropTypes.number,
     height: PropTypes.number,
     dark: PropTypes.bool,
   };
 
   render() {
-    const { dark, height, width, size } = this.props;
+    const { dark, height, width } = this.props;
     return (
       <svg
         className={cx("Icon", { "text-brand": !dark }, { "text-white": dark })}
         viewBox="0 0 66 85"
-        width={width || size}
-        height={height || size}
+        width={width}
+        height={height}
```

Comment:
Are there any places we don't set height? Is it worth having a default here or does the viewbox dimension take effect if unset?

Reply:
Good question. I'll double check that everything still works ok if you don't specify any props.